examples/jsm/deprecated/Geometry.js

```diff
@@ -18,48 +18,48 @@ const _m1 = new Matrix4();
 const _obj = new Object3D();
 const _offset = new Vector3();
 
-function Geometry() {
+class Geometry extends EventDispatcher {
```

Comment:
It's not necessary to change `Geometry`.

I would in general not touch the code anymore since testing/validation is complicated now.